client/packages/system/src/Patient/PatientView/AddButton.tsx

```diff
@@ -92,16 +113,18 @@ export const AddButton: React.FC<AddButtonProps> = ({
 
   return (
     <>
```

Comment:
I'm not usually a fan of the single node inside a fragment (99% of the time it's a mistake), but this might be cleaner in this case. I'd probably go for a ternary here (`return condition ? <SplitButton> : null`), but you decide which you think is tidier.